x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/actions_log_with_rule_toggle.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFilterButton } from '@elastic/eui';
+import React, { useCallback } from 'react';
+import { useIsExperimentalFeatureEnabled } from '../../../../common/hooks/use_experimental_features';
+import { useActionHistoryUrlParams } from './use_action_history_url_params';
+import { FILTER_NAMES } from '../translations';
+
+interface ActionsLogWithRuleToggleProps {
+  isFlyout: boolean;
+  dataTestSubj?: string;
+}
+
+const ActionsLogWithRuleToggleContent = ({
```

Comment:
Consider wrapping this with a memo here instead of doing it later. Also, consider adding a display name. It's easier to search and debug that way. One has to look for `ActionsLogWithRuleToggleContent` instead of `ActionsLogWithRuleToggle`